libs/gi/ui/src/hooks/useTitle.tsx

```diff
@@ -1,11 +1,15 @@
 import { useEffect } from 'react'
 import { useTranslation } from 'react-i18next'
+import { useMatch } from 'react-router-dom'
 
-export function useTitle(title: string) {
+export function useTitle(title?: string) {
   const { t } = useTranslation('ui')
+  const match = useMatch({ path: '/:page/*' })
   const pageTitle = t('pageTitle')
+  const displayTitle =
+    title ?? (match?.params.page && t(`tabs.${match?.params.page}`))
```

Comment:
```suggestion
    title ?? (match?.params.page && t(`tabs.${match.params.page}`))
```

Biggest smallest nit ever